pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/kit/KitFinalSentBaseUseCase.java

```diff
@@ -38,6 +38,7 @@ protected void triggerEvents(String kit, KitRequestShipping kitRequestShipping)
                 1);
         if (kitDDPNotification != null) {
             TransactionWrapper.inTransaction(conn -> {
+                // TODO: this should not be in a DB transaction since it makes a call to an external service -DC
                 EventUtil.triggerDDP(conn, kitDDPNotification);
```

Comment:
As far as I can tell, this method exists to help send off notification emails.  I have run a few queries in prod and I think we can remove this feature after talking to Eirian to confirm she has no use for these emails.  She is the only one I see getting emails in response to this scan.  There is also a configuration for testboston to receive emails, but since that study stopped sending kits over a year ago and will not restart, we can safely ignore TestBoston's needs.

Reply:
Reviewing the data a bit more, there are zero rows returned from our sendgrid logs for Eirian:

```select * from pepperapisprod.sendgrid_event where email = 'redacted'```

Reply:
Do we need another ticket to remove this?
If you provide some details I can probably create it. Not sure I have enough context at this point.